learn/get-started/create-account.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Create an Account
+---
+
+# Create an Account
+
+The first thing you’ll need to do anything on Stellar is an account. Accounts hold all your money inside Stellar and allow you to send and receive payments—in fact, pretty much everything in Stellar is in some way tied to an account.
+
+Every Stellar account has a pair of public and private keys. Stellar uses public key cryptography to ensure that every transaction is secure. The private key is a secret piece of information that proves you own the account. You should never share your private key with anyone. It’s kind of like the combination to a lock—anyone who knows the combination can open the lock. In the same way, anyone who knows your account’s secret key can control your account. The public key is how other people identify your account and verify that you authorized a transaction.
+[TODO: decide on whether we're using public/private or public/secret for keypairs. This doc currently uses both private and secret.]
```

Comment:
It's because `seed` is a deprecated term and we haven't taken the time to fix those.

We *aspire* to use "public key" and "secret key" (we had a big conversation about this a while back).

In practice though, we haven't updated a lot of our code and some stuff even refers to "address".